src/connectors/sort-by/connectSortBy.js

```diff
@@ -101,7 +100,7 @@ export default function connectSortBy(renderFn, unmountFn) {
     return {
       init({ helper, instantSearchInstance }) {
         const currentIndex = helper.getIndex();
-        const isIndexInList = find(items, item => item.value === currentIndex);
+        const isIndexInList = items.find(item => item.value === currentIndex);
```

Comment:
items can be undefined here?

Reply:
actually no it can't there's an error thrown earlier

Reply:
We throw an error for falsy value not for arrays. We should enforce the condition.